src/components/Elements/ListItem.tsx

```diff
@@ -25,22 +25,22 @@ const ListItem: React.FC<ItemProps> = (props: ItemProps) => {
     return (
         <TouchableOpacity onPress={() => onPress(id)} key={index} onLongPress={() => onLongPress(id)}>
            <Card>
-           <View style={styles.itemContainer}>
-               <View style={styles.itemTextBlock}>
-                   <Text style={styles.header}>
-                       {headerText}
-                   </Text>
-                   <Text style={styles.subheader}>
-                       {subheaderText}
-                   </Text>
+               <View style={styles.itemContainer}>
```

Comment:
I haven't changed anything here but my editor added an indentation

Reply:
You have got smart editor ))))